doc/phpactor.txt

```diff
@@ -191,20 +191,22 @@ Phpactor does not assume any mappings automatically, the following mappings
 are available for you to copy:
 >
 
-  nmap <Leader>u :PhpactorImportClass<CR>
-  nmap <Leader>ua :PhpactorImportAllClasses<CR>
-  nmap <Leader>mm :PhpactorContextMenu()<CR>
-  nmap <Leader>nn :PhpactorNavigate()<CR>
-  nmap <Leader>oo :PhpactorGotoDefinition()<CR>
-  nmap <Leader>oh :PhpactorGotoDefinitionHsplit()<CR>
-  nmap <Leader>ov :PhpactorGotoDefinitionVsplit()<CR>
-  nmap <Leader>ot :PhpactorGotoDefinitionTab()<CR>
-  nmap <Leader>K :PhpactorHover()<CR>
-  nmap <Leader>tt :PhpactorTransform()<CR>
-  nmap <Leader>cc :PhpactorClassNew()<CR>
-  nmap <silent><Leader>ee :PhpactorExtractExpression(v:false)<CR>
-  vmap <silent><Leader>ee :<C-U>PhpactorExtractExpression(v:true)<CR>
-  vmap <silent><Leader>em :<C-U>PhpactorExtractMethod()<CR>
+  augroup PhpactorMappings
```

Comment:
What's the advantage of having an `augroup` ?

Reply:
When the file is reloaded `autocmd!` will delete all autocmd, so they are not declare twiced.